Transfer/Sources/CreateTransferController.swift

```diff
@@ -280,14 +299,14 @@ extension CreateTransferController: CreateTransferView {
         if let footerView = tableView.footerView(forSection: section.rawValue) as? TransferTableViewFooterView {
             footerView.footerLabel.attributedText = getAttributedFooterText(for: section.rawValue)
         } else {
-            tableView.reloadSections(IndexSet(integer: section.rawValue), with: .none)
+            tableView.reloadData()
```

Comment:
Why reload all sections? Also cleanup the code related to footer. Now footer is just going to be error message. Previosuly Available Balance was shown as footer too. So remove that unnecessary logic